### PR TITLE
PointsAbove and PointsBellow aggregations

### PR DESCRIPTION
**Pull request by denissytenkov**
This pull request creates 2 additional aggregations

1. PointsAbove aggregation removes all points from a selected series which are less or equal specified number k.

2. PointsBellow aggregation removes all points from a selected series which are greater or equal specified number k.

### CODE REVIEW

**Comment by mykolasmith** [comment]
@denissytenkov Looks great! I've tested it and confirmed everything is working as expected. Could you please just make the one change I've commented and squash it into a single commit and we'll be good to get this merged.

File: aggregation/simple/src/main/java/com/spotify/heroic/aggregation/simple/FilterPointsKThresholdStrategy.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+    @Override
+    public MetricCollection apply(MetricCollection metrics) {
+        return MetricCollection.build(
+            MetricType.POINT,
+            filterBellow(metrics.getDataAs(Point.class))
+        );
+    }
+
+    private List<Point> filterBellow(List<Point> points) {
```
Comment (mykolasmith) [inline aggregation/simple/src/main/java/com/spotify/heroic/aggregation/simple/FilterPointsKThresholdStrategy.java]: Typo. Should be "filterBelow"

**Comment by denissytenkov** [comment]
Fixed

**Comment by mykolasmith** [comment]
Thanks! Let's schedule a follow up sometime this week to discuss and debrief.

**Pull request merged**